pkg/arvo/sys/zuse.hoon

```diff
@@ -75,10 +75,6 @@
 ++  life  @ud                                           ::  ship key revision
 ++  rift  @ud                                           ::  ship continuity
 ++  mime  {p/mite q/octs}                               ::  mimetyped data
-::
-::
-::    TODO: Rename to +mime once the current +mime and +mite are gone. The
-::
```

Comment:
`$mite` is defined in hoon.hoon, and `TODO` comments are no way to re-design data structures.